delfin/drivers/ibm/storwize_svc/ssh_handler.py

```diff
@@ -110,7 +154,7 @@ def parse_alert(alert):
     def login(self):
         try:
             with self.ssh_pool.item() as ssh:
-                result = SSHHandler.do_exec('lssystem', ssh)
+                result = SSHHandler.do_exec('ls', ssh)
```

Comment:
It is recommended to use a special command that only exist in IBM system, and Why should we change `lssystem` to `ls` ?

Reply:
done